locale/fr/LC_MESSAGES/loris.po

```diff
@@ -0,0 +1,349 @@
+msgid "Clinical"
+msgstr "Clinique"
+
+#: modules/dicom_archive/php/module.class.inc:115
+#: modules/imaging_browser/php/module.class.inc:40
+#: modules/imaging_qc/php/module.class.inc:50
+#: modules/imaging_uploader/php/module.class.inc:54
+#: modules/mri_violations/php/module.class.inc:53
+msgid "Imaging"
+msgstr "Imagerie"
+
+#: modules/candidate_list/php/module.class.inc:76
+#: modules/new_profile/php/module.class.inc:50
+msgid "Candidate"
+msgstr "Candidat"
+
+#: modules/genomic_browser/php/module.class.inc:53
+msgid "Genomics"
+msgstr "Génomique"
+
+#: modules/electrophysiology_browser/php/module.class.inc:54
+msgid "Electrophysiology"
+msgstr "Électrophysiologie"
+
+#: modules/dataquery/php/module.class.inc:33
+#: modules/dqt/php/module.class.inc:49
+#: modules/publication/php/module.class.inc:55
+#: modules/statistics/php/module.class.inc:49
+msgid "Reports"
+msgstr "Rapports"
+
+# Common loris terms. Consider moving these to their own textdomain?
+msgid "TimePoint"
+msgstr "Temps d'acquisition"
+
+# Common select option labels
+msgid "Yes"
+msgstr "Oui"
+
+msgid "No"
+msgstr "Non"
+
+msgid "Create"
+msgstr "Créer"
+
+msgid "N/A"
+msgstr "N/A"
+
+# Filters
+msgid "Selection Filter"
+msgstr "Filtre de sélection"
+
+msgid "Show Filters"
+msgstr "Afficher les filtres"
+
+msgid "Hide Filters"
+msgstr "Masquer les filtres"
+
+msgid "Clear Filters"
+msgstr "Effacer les filtres"
+
+msgid "Show Advanced Filters"
+msgstr "Afficher les filtres avancés"
+
+msgid "Hide Advanced Filters"
+msgstr "Masquer les filtres avancés"
+
+# Common swal labels
+msgid "Cancel"
+msgstr "Annuler"
+
+msgid "Success!"
+msgstr "Réussi !"
+
+# Common candidate terms
+msgid "PSCID"
+msgstr "PSCID"
+
+msgid "DCCID"
+msgstr "DCCID"
+
+#, fuzzy
+msgid "Visit Label"
+msgstr "Nom de visite"
+
+msgid "Site"
+msgstr "Site"
+
+msgid "Module"
+msgstr "Module"
+
+msgid "Project"
+msgstr "Projet"
+
+#, fuzzy
+msgid "Candidate Registration Project"
+msgstr "Projet d'inscription des candidats"
+
+#, fuzzy
+msgid "Timepoint Project"
+msgstr "Temps d’acquisition du projet"
+
+msgid "Cohort"
+msgid_plural "Cohorts"
+msgstr[0] "Cohorte"
+msgstr[1] "Cohortes"
+
+msgid "Session"
+msgstr "Session"
+
+msgid "Date of registration"
+msgstr "Date d'inscription"
+
+msgid "Participant Status"
+msgstr "Statut du participant"
+
+msgid "DoB"
+msgstr "DdN"
+
+msgid "Sex"
+msgstr "Sexe"
+
+msgid "Feedback"
+msgstr "Commentaires"
+
+msgid "Scans"
+msgstr "Scans"
+
+#, fuzzy
+msgid "Stage"
+msgstr "Stade"
+
+msgid "Send To DCC"
+msgstr "Envoyer au CCD"
+
+msgid "Sent To DCC"
+msgstr "Envoyé au CCD"
+
+msgid "Reverse Send To DCC"
+msgstr "Annuler l'envoi vers la CCD"
+
+msgid "Date of Birth"
+msgstr "Date de naissance"
+
+msgid "Visits"
+msgstr "Visites"
+
+msgid "An error occured while loading the page."
+msgstr "Une erreur s'est produite lors du chargement de la page."
+
+#, fuzzy
+msgid "Entity Type"
+msgstr "Type d'entité"
+
+#, fuzzy
+msgid "Scan Done"
+msgstr "Scan complété"
```

Comment:
Past tense matches the English "Done" but would "Scan complét" make more sense given the contexts it's used in the frontend?

@AdamAlAttrach @charliehenrib